### DISCUSSION TITLE
Theme request: Night Owl, inspired by https://vscodethemes.com/e/sdras.night-owl/night-owl?language=javascript

### DISCUSSION THREAD

**Discussion by brunoakamere** [node-red-contrib-themes/theme-collection#487] [Theme request] [answered]
### Details

It would be great to have night owl themes, inspired by https://vscodethemes.com/e/sdras.night-owl/night-owl?language=javascript.

I think it looks really clean.

**Comment by bonanitech** [comment]
Thanks for the suggestion, @brunoakamere.

I tried it once but didn't like the result because it uses a single color for backgrounds.

<img width="960" alt="SCR-20240725-pxnn" src="https://github.com/user-attachments/assets/b45ffd35-fa52-42e5-9e8f-9dd72149634b">

What do you think?

**Reply by bonanitech** [reply]
@brunoakamere, I improved it a little. Please share your thoughts about it.

<img width="1920" alt="image" src="https://github.com/user-attachments/assets/943fd867-69c9-40e7-ae4b-896b452252d0">

**Reply by brunoakamere** [reply]
I like the second look better.
I think it would also be great to have different color shades between the flow being focused on and the rest.

**Reply by bonanitech** [reply]
<img width="1920" alt="image" src="https://github.com/user-attachments/assets/b1f42fca-87ae-407e-9486-9bb9c345e09b">

**Reply by bonanitech** [reply, marked as answer]
I think it's ready. It will be released alongside Node-RED v4.1. Again, thanks for the suggestion.

<img width="1920" alt="image" src="https://github.com/user-attachments/assets/6b2dd420-280e-407e-88c0-0f4de15fd472">
